src/EventListener/Framework/ValidateInstallationListener.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/**
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\EventListener\Framework;
+
+use Contao\CoreBundle\Adapter\ConfigAdapter;
+use Contao\CoreBundle\Event\ContaoFrameworkInitializeEvent;
+use Contao\CoreBundle\Exception\IncompleteInstallationException;
+use Contao\CoreBundle\Exception\InsecureInstallationException;
+
+/**
+ * Validates the Installation
+ *
+ * @author Christian Schiffler <https://github.com/discordier>
+ * @author Yanick Witschi <https://github.com/toflar>
+ * @author Leo Feyer <https://github.com/leofeyer>
+ * @author Dominik Tomasi <https://github.com/dtomasi>
+ */
+class ValidateInstallationListener
```

Comment:
Its depending on each other ... This is the actual state of `InitializeSystemListener` which is splitted in this PR.